docs/source/release_notes/36_0.rst

```diff
@@ -1,12 +1,12 @@
 =======
-36.0lts
+36.0LTS
```

Comment:
The official name was `36.0lts` but that is incorrect version according to PEP. So new LTS releases will be just the version and the `LTS` would only be present in the name: https://trello.com/c/WI1Nwpdb/708-evaluate-version-numbering-scheme-for-lts-releases

So if I understand the result correctly, it should be `36.0 - LTS` (or `36.0 (LTS)` but I really hate the format of that)